packages/v4/src/content/design-guidelines/components/menu/menu.md

```diff
@@ -61,6 +61,14 @@ Favoriting or other optional actions can be associated with any menu item. In th
 
 Any action that can be represented as an icon button can be placed in one or more menu items. In this case, the favoriting action is used to mark an item as a favorite.
 
+### Drilldown menu
+<img src="./img/drill-down-menu.png" alt="drilldown menu" width="935"/>
+
+Use a drilldown menu when data is structured into levels and includes a long list of options. When the parent with children is selected, the list is replaced with children items. A header displays the name of a parent with the option going one level back. 
```

Comment:
Small edit: replaced with **the** children items.

Also would rephrase this and say " A header displays the name of the parent, with the option to go back one level."